src/main/scala/exercise4/Task2.scala

```diff
@@ -0,0 +1,20 @@
+package exercise4
+
+import zio.{ExitCode, URIO, ZIO}
+
+object Task2 extends zio.App {
+  /**
+   * Start a fiber: [fork] an effect to give a fiber
+   * [join]: waiting for fiber to compute value
+   * get rid of fiber by [interrupt]
+   * [zipPar] to execute actions in parallel
+   *
+   * The first computation is an interruption.
+   */
+  val app = for {
+    fiber <- (ZIO.interrupt zipPar ZIO.fail("Error")).fork
```

Comment:
It does produce the same stack trace, however`ZIO.interrupt` is a bit weird, since it just starts a zio and immediately interrupts it.
Something like ```ZIO.succeed(2).forever``` would be more sensible. That would get interrupted due to `zipPar`, since `ZIO.fail("foo")` failed.